source/include/drivers/cs_Uicr.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Author: Crownstone Team
+ * Copyright: Crownstone (https://crownstone.rocks)
+ * Date: Mar 17, 2022
+ * License: LGPLv3+, Apache License 2.0, and/or MIT (triple-licensed)
+ */
+
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <protocol/cs_UicrPacket.h>
+#include <protocol/cs_Typedefs.h>
+#include <stdbool.h>
+#include <stdint.h>
+
+/**
+ * Get the hardware board from UICR, or the default board if none is set in UICR.
```

Comment:
Get the "hardware board" is a bit sparse. Get the hardware board index? Get the hardware board version? Get the hardware board product id?